src/lib.rs

```diff
@@ -43,19 +46,6 @@ use indicatif::ProgressBar;
 pub struct Config {
     manifest_path: PathBuf,
     host: analysis::AnalysisHost,
-    assets: Vec<Asset>,
```

Comment:
If my comment wasn't clear I like everything but the global state and prefer this modularity. The build.rs improvements are a big plus.